fe/fe-core/src/main/java/com/starrocks/sql/analyzer/FunctionAnalyzer.java

```diff
@@ -34,6 +34,8 @@
 
 public class FunctionAnalyzer {
 
+    public static final int MAX_COUNTER_NUM_OF_TOP_K = 100000;
```

Comment:
Should put this definition to `fe/fe-core/src/main/java/com/starrocks/common/FeConstants.java`

Reply:
Done